packages/logs-async-reporter/logs-async-reporter.1.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+name: "logs-async-reporter"
+version: "1.0"
+authors: "Vincent Bernardoff <vb@luminar.eu.org>"
+maintainer: "Vincent Bernardoff <vb@luminar.eu.org>"
+homepage: "https://github.com/vbmithr/logs-async"
+bug-reports: "https://github.com/vbmithr/logs-async/issues"
+dev-repo: "git+https://github.com/vbmithr/logs-async"
+doc: "https://vbmithr.github.io/logs-async/doc"
+build: [ "dune" "build" "-j" jobs "-p" name "@install" ]
```

Comment:
Unimportant side-note: `"@install"` is optional (it's the default target when calling build)